index.js

```diff
@@ -455,7 +453,7 @@ function getWeightedRandomChoice(choices, weights, amountOfChoices) {
     }
 
     //iterate and find amountOfChoices choices
-    let randomNumber = Math.random();
+    let randomNumber = Math.random(); // Not cryptographically-secure
```

Comment:
Why does this have to be cryptocraphically secure? This is for distributing sponsors to people.

Reply:
Just making sure people dont use randomNumber for anything that needs to be "secure"